### ISSUE TITLE
For easier updates, can you provide tags like: major.minor

### ISSUE THREAD

**Issue by williamdes** [jbergstroem/mariadb-alpine#213]
And on each patch major.minor is updated

This also makes it possible for automatic updates using watchtower for example

**Comment by jbergstroem** [comment]
Just to clarify: aside from `latest` and `10.6.12`, you are looking for `10.6`? If yes, I like it.

**Comment by williamdes** [comment]
yes exactly, to have a latest but for a version range

**Comment by jbergstroem** [comment]
One thing to consider here is that Alpine doesn't maintain multiple versions of packages very long which means that leaning on `10.6` after the new LTS is chosen would likely make you go out of date unless there are serious critical security bumps. Also, renovatebot wouldn't track all versions of MariaDB in Alpine. I could consider adding git branches for minors and only bump for security updates which would likely happen once Alpine chooses the new LTS.

**Comment by williamdes** [comment]
do you mean that you only install the alpine MariaDB server package?
and everything will depend on alpine upstream?

**Comment by jbergstroem** [comment]
> do you mean that you only install the alpine MariaDB server package?
> and everything will depend on alpine upstream?

That is how this repository works at the moment, yes! It's not necessarily a Bad Thing; most containers relies on distributions for dependency mangement. It just makes it harder to maintain multiple release lines.

That said, MariaDB doesn't encourage using non-LTS releases outside of their lifecycle which means that there won't be multiple supported release lines.

You can get an overview of all releases here: https://mariadb.com/kb/en/mariadb-server-release-dates/

MariaDB 10.11 will become LTS which means that 10.6 and 10.11 should overlap and likely exist in parallel for Alpine Linux release lines.

**Comment by williamdes** [comment]
yeah I understand all this, some companies can not upgrade because they are too afraid or such things
but I can upgrade, I was wondering why this repository was still on 10.6
can you provide newer versions?

**Comment by jbergstroem** [comment]
> can you provide newer versions?

`10.6` is the currrent LTS version which means its the one that is packaged in Alpine. As `10.11` becomes LTS, it will find its way here.

**Comment by williamdes** [comment]
okay, so I think you could provide a lts docker tag so installations always track the lts version?

**Comment by williamdes** [comment]
by the way I do a bunch of containers at @sudo-bot so maybe you would want to join if you intend to do such a container
We have the the sponsored OSS level of docker registry: https://hub.docker.com/r/botsudo/capistrano

**Comment by jbergstroem** [comment]
> okay, so I think you could provide a lts docker tag so installations always track the lts version?

This is what the `latest` flag would provide at the moment. I think the best option – as you suggested – will be `10.6` in your case. The problem with a "LTS" tag is that the LTS tags are overlapping (both `10.6` and `10.11` would have it).

My suggestion is – for now – to stick with `latest` which is 100% guaranteed to work and track latest LTS. As `10.6` lands (probs this weekend), feel free to switch.

**Comment by williamdes** [comment]
thanks for 10.6
the thing is by using latest you really do not know what version it will be if you don't update your config for one or two years
but now I know latest is lts ^^

**Comment by jbergstroem** [comment]
> the thing is by using latest you really do not know what version it will be if you don't update your config for one or two years

Across Docker Hub I agree; there is no standardized/agreed upon best practice for `latest`.
